apps/prairielearn/src/schemas/schemas/infoQuestion.json

```diff
@@ -27,6 +27,10 @@
       "description": "The category of question (e.g., 'Vectors', 'Energy').",
       "type": "string"
     },
+    "draftVersion": {
+      "description": "The draft version number (not for final questions)",
+      "type": "integer"
+    },
```

Comment:
I don't see any reference to this elsewhere in this PR. Where is this used? Is it used?

Reply:
Removed in 562ccc7ca4238b061cc5fed79757979f8c0db35b.